analytics-api/src/analytics_api/config.py

```diff
@@ -28,20 +28,26 @@
 load_dotenv(find_dotenv())
 
 
-def get_named_config(config_name: str = 'development'):
-    """Return the configuration object based on the name.
+def get_named_config(environment: str | None) -> '_Config':
+    """
+    Retrieve a configuration object by name. Used by the Flask app factory.
 
-    :raise: KeyError: if an unknown configuration is requested
+    :param config_name: The name of the configuration.
+    :return: The requested configuration object.
+    :raises: KeyError if the requested configuration is not found.
     """
-    if config_name in ['production', 'staging', 'default']:
-        config = ProdConfig()
-    elif config_name == 'testing':
-        config = TestConfig()
-    elif config_name == 'development':
-        config = DevConfig()
-    else:
-        raise KeyError("Unknown configuration '{config_name}'")
-    return config
+    config_mapping = {
+        'development': DevConfig,
+        'default':   ProdConfig,
+        'staging':    ProdConfig,
+        'production': ProdConfig,
+        'testing':    TestConfig,
+    }
+    try:
+        print(f'Loading configuration: {environment}...')
```

Comment:
is print the logging mechanism for this application, no logging plugin or code

Reply:
Print was the logging mechanism previously used in this file. I can upgrade to `current_app.logger.debug` :)

Reply:
Nevermind; I cannot. `current_app` is only defined after the configuration is set up, and that is done in this file. I can print to STDERR